src/main/java/com/accantosystems/stratoss/vnfmdriver/driver/VNFLifecycleManagementDriver.java

```diff
@@ -93,14 +97,15 @@ public VNFLifecycleManagementDriver(AuthenticatedRestTemplateService authenticat
      * @return newly created {@link VnfInstance} record
      * @throws SOL003ResponseException if there are any errors creating the VNF instance
      */
-    public String createVnfInstance(final ResourceManagerDeploymentLocation deploymentLocation, final String createVnfRequest) throws SOL003ResponseException {
+    public String createVnfInstance(final ResourceManagerDeploymentLocation deploymentLocation, final String createVnfRequest, final String driverrequestid) throws SOL003ResponseException {
         final String url = deploymentLocation.getProperties().get(VNFM_SERVER_URL) + API_CONTEXT_ROOT + API_PREFIX_VNF_INSTANCES;
         final HttpHeaders headers = getHttpHeaders(deploymentLocation);
         headers.setContentType(MediaType.APPLICATION_JSON);
         final HttpEntity<String> requestEntity = new HttpEntity<>(createVnfRequest, headers);
-
+        UUID uuid = UUID.randomUUID();
+        LoggingUtils.logEnabledMDC(createVnfRequest, MessageType.REQUEST,MessageDirection.SENT, uuid.toString(),MediaType.APPLICATION_JSON.toString(), "http",null,driverrequestid);
```

Comment:
You can add at least url in the request protocol_metadata. This is applicable for all requests.